src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable camelcase */
+const LOAD_ROCKETS = 'SPACE_TRAVELERS/ROCKETS/LOAD_ROCKETS';
+const RESERVE_ROCKET = 'SPACE_TRAVELERS/ROCKETS/RESERVE_ROCKETS';
+const CANCEL_ROCKET = 'SPACE_TRAVELERS/ROCKETS/CANCEL_ROCKETS';
+
+const initialState = [];
+
+const selectFewerProps = (show) => {
+  const {
+    id, rocket_name, description, flickr_images,
+  } = show;
+  return {
+    id, rocket_name, description, flickr_images,
+  };
+};
+export const loadRockets = (payload) => ({
+  type: LOAD_ROCKETS,
+  payload,
+});
+
+export const reserveRockets = (payload) => ({
+  type: RESERVE_ROCKET,
+  payload,
+});
+
+export const cancelRockets = (payload) => ({
+  type: CANCEL_ROCKET,
+  payload,
+});
+
+export const loadRocketsData = () => async (dispatch) => {
+  const res = await fetch('https://api.spacexdata.com/v3/rockets');
+  const data = await res.json();
+  // eslint-disable-next-line
+  console.log(data);
+  const rocket_data = data.map(selectFewerProps);
+  dispatch(loadRockets(rocket_data));
```

Comment:
- [ ] Prefer naming your variable using the cameCase pattern. It is common in Javascript and it is part of best practice as well.